pkg/ioutils/copy.go

```diff
@@ -0,0 +1,21 @@
+package ioutils
```

Comment:
If I'll move pool there, it'll create import cycle and `pools.Copy` doesn't sound like it belongs there.